config/locales/onesky_en/rents.en.yml

```diff
@@ -1,19 +1,23 @@
-# This file is generated by onesky-rails gem and will be overwritten at the next download
-# Therefore, you should not modify this file
-# If you want to modify the translation, please do it at OneSky platform
-# If you still want to modify this file directly, please upload this file to OneSky platform after modification in order to update the translation at OneSky
-
 ---
 en:
   rents:
     form:
+      the_terms: "the terms and conditions."
+      read_terms: "Read through and accept"
+      agree_to_terms: "I accept the terms and conditions"
+      school_hours_warning: "Bookings starting or ending outside of school hours (weekdays 8-17) have to be accepted manually"
+      note: Note!
       allowed_to_overbook: "allowed to overbook up to 5 days into the future"
       cannot_edit_rent: "You can no longer edit the booking"
       council_booking: "Council booking"
       edit_user_information: "Edit user details"
       only_for_council: "can only be done on behalf of the council"
       overlaps_with: "Overlaps with:"
       terms: "Booking terms"
+    car_bans:
+      banned: "Du är spärrad från bilbokningstjänsten."
+      reason: "Anledning: "
+      for_councils: "Utskottsbokning kan fortfarande genomföras."
     index:
```

Comment:
Why have you added Swedish translations in the `en` file?

Reply:
> Why have you added Swedish translations in the `en` file?

OneSky added those, as there was no translation for it. I've added the translations now.